libraries.json

```diff
@@ -387,7 +382,7 @@
   {
     "github": "apollographql/apollo-android",
     "category": "Network",
-    "maven": "https://dl.bintray.com/apollographql/android/com/apollographql/apollo/apollo-api/"
+    "maven": "https://repo1.maven.org/maven2/com/apollographql/apollo/apollo-android-support/"
```

Comment:
it's not multiplatform artifact. but https://repo1.maven.org/maven2/com/apollographql/apollo/apollo-api/ is multiplatform

Reply:
fixed